script/release.js

```diff
@@ -195,37 +192,41 @@ async function createReleaseShasums (release) {
   const existingAssets = release.assets.filter(asset => asset.name === fileName)
   if (existingAssets.length > 0) {
     console.log(`${fileName} already exists on GitHub; deleting before creating new file.`)
-    await github.repos.deleteAsset({
+    await octokit.repos.deleteReleaseAsset({
       owner: 'electron',
       repo: targetRepo,
-      id: existingAssets[0].id
+      asset_id: existingAssets[0].id
     }).catch(err => {
       console.log(`${fail} Error deleting ${fileName} on GitHub:`, err)
     })
   }
   console.log(`Creating and uploading the release ${fileName}.`)
   const scriptPath = path.join(__dirname, 'merge-electron-checksums.py')
   const checksums = runScript(scriptPath, ['-v', pkgVersion])
+
   console.log(`${pass} Generated release SHASUMS.`)
   const filePath = await saveShaSumFile(checksums, fileName)
+
   console.log(`${pass} Created ${fileName} file.`)
-  await uploadShasumFile(filePath, fileName, release)
+  await uploadShasumFile(filePath, fileName, release.id)
+
   console.log(`${pass} Successfully uploaded ${fileName} to GitHub.`)
 }
 
-async function uploadShasumFile (filePath, fileName, release) {
-  const githubOpts = {
-    owner: 'electron',
-    repo: targetRepo,
-    id: release.id,
-    filePath,
+async function uploadShasumFile (filePath, fileName, releaseId) {
+  const uploadUrl = `https://uploads.github.com/repos/electron/${targetRepo}/releases/${releaseId}/assets{?name,label}`
```

Comment:
Why do we need a upload url here?

Reply:
https://octokit.github.io/rest.js/#api-Repos-uploadReleaseAsset it's required now